### ISSUE TITLE
[bug] Aeotec Wall Mote Quad - Scene is undefined

### ISSUE THREAD

**Issue by tommyjlong** [zwave-js/zwavejs2mqtt#223]
**Version**
1.0.0-alpha.2.51f715c
Build/Run method
- [ ] Docker
- [ ] PKG
- [x] Manually built (git clone - npm install - npm run build ) 

zwavejs2mqtt version: 1.0.0-alpha.2.51f715c

**Describe the bug**
Central Scene showing up in logs as undefined.
_Looks like this has not yet been implemented, but thought I would raise the issue_

**To Reproduce**
1. Tap button1 on Wallmote Quad
2. See error

zwavejs2mqtt log entry:
`2021-01-12 14:37:24.812 INFO ZWAVE: Node 8: value notification: 91-0-scene-001 undefined => undefined`

zwave-js log:
```
14:37:24.813 CNTRLR « [Node 008] received CentralScene notification {
                          "nodeId": 8,
                          "ccId": "Central Scene",
                          "ccCommand": "0x03",
                          "payload": "0x740001"
                      }
14:37:24.858 CNTRLR   [Node 008] treating BasicCC Set as a report
14:37:25.385 DRIVER   Dropping message because it could not be deserialized: MultilevelSwitchCCSet: 
                      deserialization not implemented
```

**Comment by blhoward2** [comment]
It’s been implemented. There was a recent bug that should be fixed by https://github.com/zwave-js/zwavejs2mqtt/pull/212

**Comment by robertsLando** [comment]
@tommyjlong Are you able to build a docker container from master branch of both projects?

**Comment by robertsLando** [comment]
https://github.com/zwave-js/zwavejs2mqtt/tree/master/docker#building-a-container-using-dockerfilecontrib

**Comment by tommyjlong** [comment]
I have been building my cloned repositories with npm.
I built today's zwavejs2mqtt with npm and tested out this fix, but it didn't work.  I checked the PR and its says the fix #212 was in lib/ZwaveClient.js and it was merged, but when I look at lib/ZwaveClient.js on my repository as well as the github master branch, I don't see the fix.
